packages/nice-grpc-prometheus/README.md

```diff
@@ -89,3 +89,54 @@ Metrics that correspond to finished calls have extra label:
 
 [npm-image]: https://badge.fury.io/js/nice-grpc-prometheus.svg
 [npm-url]: https://badge.fury.io/js/nice-grpc-prometheus
+
+### Customization
+
+You can use your own metric instances. This can be useful for example if you
+want to use your own buckets in histograms.
+
+```ts
+import {createClientFactory} from 'nice-grpc';
+import {
+  labelNames,
+  prometheusClientMiddleware,
+  registry,
+} from 'nice-grpc-prometheus';
+import {Histogram} from 'prom-client';
+
+const clientHandlingSecondsMetric = new Histogram({
+  registers: [registry],
```

Comment:
The example here reuses the lib's registry, which means that you won't be able to use the same metric names. So to change the buckets you would have to use custom names, which does not sound convenient.

The tests use a separate registry, which I think is the correct way to go. But we need an example to mention this and also show how to merge the registries.

Reply:
You're right. It's updated now.